RELEASE_HOWTO.md

```diff
@@ -0,0 +1,21 @@
+# How to cut a new release
+
+1.  Make sure you are on the right branch, usually master
+2.  Make sure everything that needs to be committed is committed
+3.  Pick a version number.
+    * Bugfixes: final digit
+    * New features visible to user: middle digit
+    * New way to deploy: middle digit
+4.  Update the version number in the file `package.json` first, easy to forget.
+    It's close to the top of the file.
+5.  Run `npm install` to upgrade the `package-lock.json`-file
+6.  Update the changelog with `towncrier build --version yournewversionnumber`.
+    Feel free to massage the generated text!
+7.  Commit the changes from steps 4 to 6 in the same commit.
+8.  Tag the commit made in step 7 with an annotated tag with the new version
+    number. "Annotated" means it needs the `-a` flag, and a message. You can
+    give the message in the same go with the `-m`-flag. Sum up the most
+    important thing in the release in the message.
```

Comment:
Here I would love an example, because from the text to me it would not be clear how to use the `-a` or `-m` flags

Reply:
I won't be adding any more examples, because I'm the one doing the releases right now and I need just hints. Great detail can be added later, in other PRs.